l10n_it_account_tax_kind/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,11 @@
+**Italiano**
+
+Questo modulo estende la funzionalità delle imposte.
+
+Con questo modulo è possibile settare un tipo di esenzione per ogni imposta.
```

Comment:
Settare è un anglicismo che si è imposto nella terminologia tecnica informatica e può andare bene in ambito tecnico a livello informale.
Dato che in questo caso si tratta di documentazione formale, che oltretutto è anche rivolta a un pubblico funzionale, userei "impostare".
Se non ti piace la ripetizione impostare/imposta presente nella frase potresti usare il termine "assegnare".